cognite/powerops/utils/serialization.py

```diff
@@ -207,8 +207,12 @@ def load_yaml(
         )
     output = CSafeLoader(data).get_data()
     if expected_return_type == "dict" and not isinstance(output, dict):
+        if not output:
+            return {}
         raise ValueError(f"Expected a dictionary, got {type(output)}")
     if expected_return_type == "list" and not isinstance(output, list):
+        if not output:
+            return []

```

Comment:
**suggestion:** I'd add a log message before the return to just log a warning that the yaml was empty

Reply:
I don't really agree with how `warnings` are used here, the pre-existing warning in this function should be a log message in my opinion 😅 But for now what you did is fine since I've already seen other places where there's a big discrepancy and inconsistency in the SDK with how and where log/warning/print is used. So there's another task to be done in the clean-up of the SDK before v1 release